platform/linux-generic/odp_packet_io.c

```diff
@@ -652,6 +652,13 @@ int odp_pktio_start(odp_pktio_t hdl)
 		_ODP_ERR("Already started\n");
 		return -1;
 	}
+
+	if (entry->state == PKTIO_STATE_STOP_PENDING) {
+		unlock_entry(entry);
+		_ODP_ERR("Scheduled pktio stop pending. Retry after calling odp_schedule().\n");
```

Comment:
"Scheduled pktio stop pending" is probably enough here, otherwise OK.

Reply:
Fixed in V2.